web-app/src/services/auth.ts

```diff
@@ -26,3 +26,28 @@ export const verifyUserCode = async ({ email, verificationCode }: VerifyUserCode
   });
   return { success: result.status === 204 };
 };
+
+interface SignInUserProps {
+  email: string;
+  password: string;
+}
+
+export const signInUser = async ({ email, password }: SignInUserProps) => {
+  try {
+    const result = await fetch(`${baseApiUrl}auth/sign-in`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ userEmail: email, userPassword: password })
+    });
+
+    if (!result.ok) {
+      const errorMessage = await result.text();
+      throw new Error(`Failed to sign in: ${errorMessage}`);
+    }
+
+    return { success: true };
+  } catch (error) {
+    console.error('Error signing in:', error);
+    return { success: false, message: 'Failed to sign in. Please try again later.' };
+  }
+};
```

Comment:
This one is not quite right
The sign in endpoints returns auth tokens that we need to store

Reply:
But no worries on this one, I can work on it after